android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/login/SuggestStrongPasswordBar.kt

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.prompts.login
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.util.AttributeSet
+import android.view.View
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.withStyledAttributes
+import androidx.core.view.isVisible
+import androidx.core.widget.TextViewCompat
+import mozilla.components.feature.prompts.R
+import mozilla.components.feature.prompts.concept.SelectablePromptView
+import mozilla.components.feature.prompts.concept.SimplePromptView
+
+/**
+ * A customizable multiple login selection bar implementing [SelectablePromptView].
```

Comment:
Could this be more specific? It looks the same as `LoginSelectBar` description, which is a little ambiguous